backend/Repositories/PlayersRepository.cs

```diff
@@ -14,36 +12,34 @@ public interface IPlayersRepository
         void Delete(string code);
     }
 
-    public class PlayersRepository: IPlayersRepository
+    public class PlayersRepository : IPlayersRepository
     {
         public List<Player> GetAll()
         {
-            return this.data;
+            return _data;
         }
 
         public Player Get(string code)
         {
-            return this.data.Find(i => i.Code == code);
+            return _data.First(i => i.Code == code);
```

Comment:
While `Find` was Ok, that is a `List<T>` method. 
The `First` is a LINQ method which works more generically on any `IEnumerable<T>` / `IQueryable<T>`.
This will be preferred if we use EF for example.

Note the difference between First and FirstOrDefault.